google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -282,9 +282,11 @@ private File doStageCredentials(String googleUsername) {
 
   @Override
   public void deleteCredentials() {
-    if (credentialsPath != null && credentialsPath.exists()) {
-      if (!credentialsPath.delete()) {
-        logger.warn("failed to delete credential file expected at " + credentialsPath.getPath());
+    if (credentialsPath != null && Files.exists(credentialsPath)) {
+      try {
+        Files.delete(credentialsPath);
+      } catch (IOException ioe) {
```

Comment:
won't this throw other types of exceptions too? You are covered on `NoSuchFileException` because of of the `exists` check, but what about all the others?